app/uk/gov/hmrc/exports/services/mapping/declaration/DeclarationBuilder.scala

```diff
@@ -89,8 +93,35 @@ class DeclarationBuilder @Inject() (
     identificationBuilder.buildThenAdd(declarationId, declaration)
     submitterBuilder.buildThenAdd(eori, declaration)
     amendmentBuilder.buildThenAdd(changeReason, declaration)
-    additionalInformationBuilder.buildThenAdd(statementDescription, declaration)
+    buildThenAddAdditionalInformation(statementDescription, declaration)
 
     declaration
   }
+
+  private def buildThenAddAdditionalInformation(statementDescription: String, declaration: Declaration): Unit = {
+    val additionalInformation = new Declaration.AdditionalInformation()
+    val statementDescriptionTextType = new AdditionalInformationStatementDescriptionTextType()
+    statementDescriptionTextType.setValue(statementDescription)
+
+    val statementTypeCode = new AdditionalInformationStatementTypeCodeType()
+    statementTypeCode.setValue("AES")
+
+    val pointer1 = new Pointer()
+    pointer1.setSequenceNumeric(new java.math.BigDecimal(1))
+    val pointerDocumentSectionCodeType1 = new PointerDocumentSectionCodeType()
+    pointerDocumentSectionCodeType1.setValue("42A")
+    pointer1.setDocumentSectionCode(pointerDocumentSectionCodeType1)
+
+    val pointer2 = new Pointer()
+    val pointerDocumentSectionCodeType2 = new PointerDocumentSectionCodeType()
+    pointerDocumentSectionCodeType2.setValue("06A")
+    pointer2.setDocumentSectionCode(pointerDocumentSectionCodeType2)
+
+    additionalInformation.getPointer.add(pointer1)
+    additionalInformation.getPointer.add(pointer2)
+
+    additionalInformation.setStatementDescription(statementDescriptionTextType)
+    additionalInformation.setStatementTypeCode(statementTypeCode)
+    declaration.getAdditionalInformation.add(additionalInformation)
```

Comment:
Nit, would have kept this code in its own separate builder class :slightly_smiling_face: